docs/blogs/zoda.md

```diff
@@ -0,0 +1,306 @@
+---
+title: "Optimizing Block Dispersal with Coding"
+description: "In which we describe how we use coding schemes, and ZODA to distribute blocks efficiently"
+date: "October 15th, 2025"
+author: "Lucas Meier"
+katex: true
+---
+
+A crucial part of blockchain consensus is distributing the block among participants.
+You can come to consensus over a mere fingerprint of the block---a hash for example---
+but doing anything interesting with the block, like processing transactions, requires
+the data.
+
+We've invested some effort in this part of the Commonware stack recently, and I'd
+like to share some fruits of that effort in this post.
+As an outline, we'll cover:
+- data dissemination, naively,
+- how to increase efficiency with coding,
+- and, finally, how to get quicker guarantees about the data with [ZODA](https://eprint.iacr.org/2025/034).
+
+In the future post could cover lower-level details of our implementation, like
+the field we use for Reed-Solomon coding, and the optimizations needed for
+fast fourier transforms, but this one will stick to an overview.
+
+## Naive Dissemination
+
+We assume a _leader_ has some data, $D$ bytes worth, to be disseminated.
+They want $m$ followers to receive it.
+The simplest approach is to have the leader send the data to every follower.
+The leader's transmission cost is $m \cdot D$ bytes, and the followers' cost
+is $0$ bytes, since they send nothing.
```

Comment:
Do we not count ingress bandwidth?

Seems there are several formulas we could use like `max(ingress, outgress)` or `ingress + outgress` but you seem to just assume `outgress` as the transmission cost